modules/special-rules/pages/flak.adoc

```diff
@@ -1,2 +1,62 @@
 = Flak
 
+These units are specialised to shoot down enemy flyers and drop pods.
+Some use very sophisticated targeting devices and fast, accurate weapons systems.
+Others simply fill the air with so many munitions that they can hardly fail to get some hits.
+
+---
+
+{flak} units can attack flyers in one of two different ways, depending on what orders you give to their detachment.
+
+[IMPORTANT]
+.Flak and drop pods
+====
+{flak} units may fire on drop pods as if they were flyers, against the Armour value of the transported unit.
+====
+
+== Flak orders
+For optimum effectiveness, {flak} units and their detachment need to be at a suitable state of readiness.
+We represent this state with special orders called Flak.
+
+. If you wish to place any detachments on Flak orders, you must do this in the Start phase.
```

Comment:
```suggestion
. If you wish to place any detachments on Flak orders then you must do this in the Start phase.
```
I've just suggested insertion of 'then' in place of the comma because I think this is more correct (continuation of the same clause). I should've done this in in the first place.
Start phase is correct of course, nice catch (I thought I'd done that - must've gotten lost in the mix of other things going on).